buggy-queries.txt

```diff
@@ -0,0 +1,2 @@
+what is the average salary of employees where name is not vinh
+how many jobs where name is not vinh
```

Comment:
this should not work! a job does not have employees related to it. Imagine if you said 

    how many jobs are there where name is vinh

and there are two vinhs. The answer would be 2 or ERROR. 2 makes no sense! There are two employees with name vinh. You could count distinct jobs from those employees if you wanted to know how many jobs have AN employee with that name. Error is the only reasonable output.